modules/CvdiGeneBurden.py

```diff
@@ -56,6 +55,8 @@ def main(
         associations_df.merge(p_value_cutoff_df, left_on="method_name", right_on="Mask")
         .drop("Mask", axis=1)
         .drop_duplicates()
+        # Dropping rows with no odds ratio:
+        .astype({"OR [95%CI]": str})
```

Comment:
Will this get rid of rows where OR is `NA [NA; NA]`? Were those the problematic ones?

Reply:
I think the NaN will be cast to "NaN", which then will not be matched by any of the regexes, so all derived columns are null.